### PR TITLE
chore: add {Darma Al Gani} as 2024 Academician

### PR DESCRIPTION
**Pull request by Daarma-IC**

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -80,8 +80,11 @@ Welcome all of you future star candidates who want to strive to attend the acade
 - [Umar Zaki Gunawan](https://github.com/marzkigun27) [UZY]
   > As an academician.
 - [Rafhan Mazaya Fathurrahman](https://github.com/fhanyuh) [RAF]
-  > As an academician.  
+  > As an academician.
   > Website link -> (https://basic-frontend-rose.vercel.app/)
+- [Darma Al Gani](https://github.com/Daarma_IC) [GAN]
+  > As an academician. 
+
```
Comment (fakhrip) [inline Readme.md:87]: Please remove this newline here as its not needed

**Review by fakhrip — CHANGES_REQUESTED** [review]
Please dont change other lines, since it will change the formatting as well (see the one with Rafhan Mazaya)

Change only what you need to change which in this case is just to **add** your name and stuff 👍

Please fix it once again, thanks!

**Review by Daarma-IC — COMMENTED** [review]
Deleted line 86,87

**Comment by fakhrip** [comment]
You have just added your personal web codes here again 😅 , okay so good thing is that you might be learning a lot from all these here, so the thing is that you would want to push your personal web codes to another repo of yours instead of the fork of this repo

Please remove it along with the fix that i have mentioned previously, and also please click on this re-request review icon beside the red icon thingy when youre done, thanks!
image

If you have any question, i suggest you to ask to the tentors or mentors to understand what is the right thing to do here 👍

**Comment by Daarma-IC** [comment]
sorry about it and promise me, and now im already deleted what i make change before, thanks for your pleasure

**Review by Daarma-IC — COMMENTED** [review]
Already create and update the newest

**Pull request closed without merging**